src/resources/assets/js/createContainer.vue

```diff
@@ -626,8 +643,44 @@ export default {
                     info: false,
                 });
             });
-        }
-    }, 
+        },
+        getHightWidth(file, path, callback) {
+            let reader = new FileReader();
+            reader.onload = function (event) {
+                let rightSize = true;
+                let buffer = event.target.result;
+                let threshold = parseInt(document.getElementById("imageThreshold").value);
+                let ifds = [];
+
+                try {
+                    ifds = UTIF.decode(buffer);
+                } catch (error) {
+                    console.warn("UTIF.decode() failed.");
+                    callback(true, path); // decoding failed, add file
+                    return;
+                }
+                if (ifds.length > 0) {
+                    let width = ifds[0].t256[0];
+                    let height = ifds[0].t257[0];
+                    if (width < threshold || height < threshold) {
+                        rightSize = false;
+                    }
+                    callback(rightSize, path);
+                } else {
+                    callback(true, path);
+                }
+            };
+            reader.onerror = function (e) {
+                console.error("FileReader error:", e);
+                callback(true, path);
+            };
+            reader.readAsArrayBuffer(file);
+        },
+        updateRejectedTiff() {
+            this.hasRejectedTiff = true
```

Comment:
Please watch your coding style.

```suggestion
            this.hasRejectedTiff = true;
```